src/mcpbr/docker_env.py

```
@@ -611,19 +631,60 @@ async def _copy_repo_to_workspace(self, env: TaskEnvironment) -> None:
+                "Workspace populated after sync retry "
+                f"(instance={env.instance_id}, files={file_count})"
             )
+            env.workdir = "/workspace"
+            return
+
+        # --- Phase 3: full copy retry ---
+        logger.warning(
+            "Workspace still empty after sync retry — re-copying from /testbed "
+            f"(instance={env.instance_id})"
+        )
+        exit_code, stdout, stderr = await env.exec_command(
```

Ruff (0.14.14)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)